run-ide-install-host-deps.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+AskContinue() {
+  read -p "Continue (Y/n)?" answer
+  if [ "$answer" != "${answer//[nN]/x}" ]; then
+    exit
+  fi
+}
+
+echo "You are running flatpaked TheIDE for the first time!"
+echo "It is highly recommended that you install all required dependencies on your host operating system."
```

Comment:
Why do you need dependencies on host? You can make sdk the runtime or launch the flatpak with --devel.

Reply:
Hello @bbhtt, these dependencies are need to compile U++ applications on host machine directly from TheIDE (sandboxed app). Without them there will be compilation errors.

--devel is not an option here. We do not want to debug TheIDE itself. We want to compile completely new apps from the sandbox app.